src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+.form {
+  margin: 20px auto 0;
+  justify-content: center;
+  display: flex;
+  width: 100%;
+}
+
+.form-big {
+  height: 70px;
+  line-height: 22px;
+}
+
+.form-small {
+  height: 42px;
+}
+
+.form-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.form-label::after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.form-label-big::after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.form-label-small::after {
+  background-image: url(./images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.form-input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgb(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.form-input-big {
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.form-input-small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.form-input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.form-input:focus {
+  color: #3d4e61;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgb(61, 78, 97, 0.1);
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
```

Comment:
remove duplicate style, you already have the same ones `.form-input`. Please check your code and improve according this comment

```suggestion
```